news/2019-07-29-project-loved-week-of-july-28th.md

```diff
@@ -0,0 +1,215 @@
+
+---
+
+[![](/wiki/shared/news/2019-07-29-project-loved-week-of-july-28th/mania/1-uta.jpg)](https://osu.ppy.sh/community/forums/topics/940578)
+
+[Imperial Circus Dead Decadence - Uta](https://osu.ppy.sh/beatmapsets/981714#mania) by [Evening](https://osu.ppy.sh/users/2193881) and [XeoStyle](https://osu.ppy.sh/users/3377280)  
+140 BPM, 8:48 | [4K] 4.74★, [4K] 5.64★, [4K] 5.77★
+
+"Return of the king" is a phrase one may use seeing this nomination to the Loved section. Evening, one of the most well-known and influential mappers to ever grace osu!mania, is back in the nominations with an even more well-known and popular song from a recently added Featured Artist—Imperial Circus Dead Decadence. 
+
+The Everlasting Duet is an exhausting mix of dense jumpstream with occasional minijacks to the percussion along with dumped-out, blazing-fast streams mixed in to intensify the screaming vocals. Not to mention that the map, fitting to the difficulty name, is a daunting near *9 minutes* in length, only serving to exemplify the extreme strain that the fast, awkward, never-ending patterns have on players.
+
+If this 9 minute beast is too much to handle, then the Bloody Hopeful Romantic difficulty or XeoStyle's Blind Immortality will definitely be more approachable. The former is similar in structure to the top difficulty, with the latter being significantly easier, both stopping at the 5 and a half minute mark to ease the stamina drain of the usual onslaught.
+
+Back when this was first released, it gained significant notoriety through the community, going as far as being used as a tiebreaker in the first edition of SOFT, osu!mania's biggest 1v1 tournament, and even now still maintains a level of relevance due in no small part to the challenge and satisfaction that the chart gives,in addition to the popularity of the song keeping people interested despite its length. This gives more than enough reason for *Uta* to be nominated for the Loved section.
+
+[![](/wiki/shared/news/2019-07-29-project-loved-week-of-july-28th/mania/2-mario-paint-time-regression-mix-for-bms.jpg)](https://osu.ppy.sh/community/forums/topics/940577)
+
+[ueotan - Mario Paint (Time Regression Mix for BMS)](https://osu.ppy.sh/beatmapsets/292994#mania) by [Fullerene-](https://osu.ppy.sh/users/2531335)  
+155 BPM, 1:29 | [4K] 4.91★, [4K] 5.68★, [4K] 6.74★
+
+There aren't many mapsets that are as notorious as Fullerene's [i Mario Paint (Time Regression Mix for BMS)[/i]. Its D-HYPER difficulty is part of one of the harder 4K dan courses, and even outside of Dan courses, D-HYPER has been a benchmark for many players for many years. D-NORMAL has also been a popular chart for lower-level players, and D-ANOTHER, while originally a joke difficulty meant to emulate a chart from BMS, has been played frequently by the very best players today.
```

Comment:
the `[i]` bbcode is incomplete here (the program will convert it to markdown for you)